packages/pharos/src/components/button/pharos-button.ts

```diff
@@ -254,23 +316,30 @@ export class PharosButton extends ScopedRegistryMixin(FocusMixin(AnchorElement))
             ping=${ifDefined(this.ping)}
             rel=${ifDefined(this.rel)}
             target=${ifDefined(this.target)}
-            aria-label=${ifDefined(this.label)}
-            aria-pressed=${ifDefined(this.pressed)}
+            aria-label=${ifDefined(a11yLabel)}
+            aria-description=${ifDefined(this.a11yDescription)}
+            aria-pressed=${ifDefined(a11yPressed)}
+            aria-expanded=${ifDefined(this.a11yExpanded)}
+            aria-haspopup=${ifDefined(this.a11yHaspopup)}
             @keyup=${this._handleKeyup}
           >
             ${this.buttonContent}
           </a>
         `
       : html`
+          <!-- @ts-ignore -->
```

Comment:
From what I recall , this was because of an issue in `aria-haspopup` with lit-analyser.

I started looking into it but couldn't reproduce the problem locally from their source. That's when I saw that despite recent work, a new package hasn't been release in almost 3 years.

I asked [here](https://github.com/runem/lit-analyzer/pull/175#issuecomment-1765306567) about that, hopefully they can clarify. If they aren't releasing new versions anymore we might have to look into migrating to something else?

Reply:
Is there a way we can ignore the single problematic attribute with an inline comment rather than the whole tag?

Reply:
Not that I found, although it wasn't an exhaustive search. Per-rule comments didn't work and in the `ts-lint-typescript` documentation it only mentions the broad rule. https://github.com/rictic/ts-lit-plugin#-support-for-ts-ignore-comments-inside-html

Reply:
May be able to check on this with `lit-analyzer@^2.0.1`.